build-stp.md

```diff
@@ -8,20 +8,23 @@ slug: build-stp
 KLEE is based on the STP constraint solver. STP does not make frequent releases, and its GitHub repository is under constant development and may be unstable.
 The instructions below are for the release 2.1.2. If you would like to use the upstream version, do not perform the checkout command `git checkout tags/2.1.2`.
 
-_Please let us know if you have successfully and extensively used KLEE with a more recent version of STP._  
+_Please let us know if you have successfully and extensively used KLEE with a more recent version of STP._
 
 STP has a few external dependencies they are listed below as an install command for Ubuntu 14.04LTS.
 
 ```bash
 sudo apt-get install cmake bison flex libboost-all-dev python perl zlib1g-dev
 ```
 
+**NOTE:** If you are using an older Linux release (e.g. Ubuntu ≤12.04), then you will have to manually install cmake 2.8.8 or newer (you can follow the instructions [here](http://cameo54321.blogspot.com/2014/02/installing-cmake-288-or-higher-on.html)).    
```

Comment:
Also you should recommend a newer version CMake version. STP requires CMake >= 3.0.2, KLEE requires >= 2.8.12